release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+==== Deprecation of `/oapi` endpoint from `oc`
+
+The usage of the `/oapi` endpoint from `oc` is being deprecated and will be
+removed in a future release. The `/oapi` endpoint was responsible for serving
+non-group {product-title} APIs and was removed in 4.1.
+
+[discrete]
+[id="ocp-4-2-deprecation-of-short-flag"]
+==== Deprecation of the `-short` flag of `oc version`
+
+The `oc version --short` flag is now deprecated. The `--short` flag printed
+default output.
+
+[discrete]
+[id="ocp-4-2-api-removal"]
+==== Removal of APIs
+
+In a future release, these APIs will be removed:
+
+* operatorsource.operators.coreos.com/v1
+* *.servicecatalog.k8s.io/v1beta1
+* *.automationbroker.io/v1alpha1
+
+[discrete]
+[id="ocp-4-2-oc-adm-migrate-commands-deprecated"]
+==== `oc adm migrate` commands
+
+The `oc adm migrate` command and all of its subcommands are now deprecated.
+
+[discrete]
+[id="ocp-4-2-PV-snapshots-deprecated"]
+==== Persistent volume snapshots
+
+Persistent volume (PV) snapshots are now deprecated.
+
+[discrete]
+[id="ocp-4-2-recycle-reclaim-policy"]
+==== Recycle reclaim policy
+
+Recycle reclaim policy is now deprecated. Dynamic provisioning is recommended.
+
+[id="ocp-4-2-bug-fixes"]
+== Bug fixes
+
+List goes here.
+
+[id="ocp-4-2-technology-preview"]
+== Technology Preview features
+
+Some features in this release are currently in Technology Preview. These
+experimental features are not intended for production use. Note the
+following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
+Features Support Scope]
+
+In the table below, features marked *TP* indicate _Technology Preview_ and
+features marked *GA* indicate _General Availability_.
+
+.Technology Preview Tracker
+[cols="4",options="header"]
+|====
+|Feature |OCP 3.11 |OCP 4.1 |OCP 4.2
+
+|Prometheus Cluster Monitoring
+|GA
+|GA
+|GA
+
+|Local Storage Persistent Volumes
+|TP
+|TP
+|TP
+
+|CRI-O for runtime pods
+|GA* footnoteref:[disclaimer, Features marked with `*` indicate delivery in a z-stream patch.]
+|GA
+|GA
+
+|Tenant Driven Snapshotting
+|TP
+|TP
+|TP
+
+|`oc` CLI Plug-ins
+|TP
+|TP
+|TP
+
+|Service Catalog
+|GA
+|GA
+|--
+
+|Template Service Broker
+|GA
+|GA
+|--
+
+|OpenShift Ansible Service Broker
+|GA
+|GA
+|--
+
+|Network Policy
+|GA
+|GA
+|GA
+
+|Multus
+|-
+|GA
+|GA
+
+|Service Catalog Initial Experience
+|GA
+|GA
+|GA
+
+|New Add Project Flow
+|GA
+|GA
+|GA
+
+|Search Catalog
+|GA
+|GA
+|GA
+
+|Cron Jobs
+|GA
+|GA
+|GA
+
+|Kubernetes Deployments
+|GA
+|GA
+|GA
+
+|StatefulSets
+|GA
+|GA
+|GA
+
+|Explicit Quota
+|GA
+|GA
+|GA
+
+|Mount Options
+|GA
+|GA
+|GA
+
+|System Containers for Docker, CRI-O
+|-
+|-
+|-
+
+|Hawkular Agent
+|-
+|-
+|-
+
+|Pod PreSets
+|-
+|-
+|-
+
+|experimental-qos-reserved
+|TP
+|TP
+|TP
+
+|Pod sysctls
+|GA
+|GA
+|GA. See xref:ocp-4-2-known-issues[Known issues] for current limitations.
+
+|Central Audit
+|GA
+|GA
+|GA
```

Comment:
Per https://coreos.slack.com/archives/CB48XQ4KZ/p1569400736447100?thread_ts=1569400604.446000&cid=CB48XQ4KZ , Central Audit in 4.1 and 4.2 is not GA